test/PerpetualPositionRouter.t.sol

```diff
@@ -31,7 +31,7 @@ contract PositionRouterTest is Test, PerpetualContracts {
       abi.encodePacked(
         uint8(openFunc),
         uint160(sqrtPriceLimitX96),
-        uint32(type(uint32).max),
+        uint32(block.timestamp),
```

Comment:
Why did you change all the test timestamps? Both values should be fine, just curious

Reply:
The idea was that this would be closer a more realistic value and would be more likely to catch a regression.